README.md

```diff
@@ -11,225 +11,165 @@ If you are a JIRA admin user, you can configure a JIRA WebHook (as shown in the
 ![JIRA WebHook Configuration Page](src/docs/sample/sample_webhook_event_selection.png)
 
 ## What formats and events it supports and what it produces
-Every integration will receive a message sent in a specific format (depending on the system it ingests) and will usually convert it into an "entity" before it reaches the Symphony platform. It will also, usually, identify the kind of message based on an "event" identifier, which varies based on the third-party system.
+Every integration will receive a message sent in a specific format (depending on the system it ingests) and will usually convert it into an internal format before it reaches the Symphony platform. It will also, usually, identify the kind of message based on an "event" identifier, which varies based on the third-party system.
 
-You can find more details about entities and the Symphony Message ML format [here](https://github.com/symphonyoss/App-Integrations-Core#the-message-ml-format).
+There are currently two implementation versions for the MessageML format, [V1](https://github.com/symphonyoss/App-Integrations-Core#the-message-ml-format) and [V2](https://symphonyoss.atlassian.net/wiki/display/WGFOS/MessageML+V2+Draft+Proposal+-+For+Discussion).
```

Comment:
I believe this paragraph is not necessary once we're removing references to MessageML v1.